### PR TITLE
CLOUDSTACK-9859: Retirement of midonet plugin (final removal)

### PR DESCRIPTION
**Pull request by rafaelweingartner**
Following the component retirement process defined in [1], a vote thread was started in [2]. The community decided to retire the Midonet plugin. This task represents the final step of the retirement, which is the removal of the plugin from CloudStacks code base.

[1] https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=68720798
[2] http://markmail.org/message/qigrtfirwnmct4hr

This closes #CLOUDSTACK-9859

### CODE REVIEW

File: setup/db/db/schema-41000to41100.sql [hunk 1]
```diff
@@ -248,3 +248,6 @@ ALTER TABLE `cloud`.`oobm` MODIFY COLUMN port VARCHAR(255);
 
 -- CLOUDSTACK-9902: Console proxy SSL toggle
 INSERT IGNORE INTO `cloud`.`configuration` (`category`, `instance`, `component`, `name`, `value`, `description`, `default_value`, `is_dynamic`) VALUES ('Console Proxy', 'DEFAULT', 'AgentManager', 'consoleproxy.sslEnabled', 'false', 'Enable SSL for console proxy', 'false', 0);
+
+-- CLOUDSTACK-9859: Retirement of midonet plugin (final removal) 
+delete from `cloud`.`configuration` where name in ('midonet.apiserver.address', 'midonet.providerrouter.id')
```
Comment (marcaurele) [inline setup/db/db/schema-41000to41100.sql]: Missing `;` at the end of the line

Comment (rafaelweingartner) [reply setup/db/db/schema-41000to41100.sql]: What a good pair of eyes!
Thanks!

**Comment by rhtyd** [comment]
LGTM, thanks @rafaelweingartner 
@blueorangutan package

**Comment by rafaelweingartner** [comment]
@marcaurele and @rhtyd can you also review this one?
https://github.com/apache/cloudstack-docs/pull/20

We have to reflect this on our docs as well

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by rhtyd** [comment]
Looks like environment issues, I'll rekick tests.
@blueorangutan test

**Comment by rhtyd** [comment]
Tests LGTM, will merge now. Thanks @rafaelweingartner

**Pull request merged**